files/zh-cn/learn/index.md

```diff
@@ -10,11 +10,12 @@ tags:
   - 前端
```

Comment:
please rebase these un-need files

Reply:
This file change for show up the CI how to check the changed files and the CI execute result. 

If you have no question for the CI config, I can cancel this file change.

Reply:
CI result example:

https://github.com/mdn/translated-content/runs/5244123027?check_suite_focus=true